orm/sequence_property_test.go

```diff
@@ -0,0 +1,71 @@
+package orm_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"pgregory.net/rapid"
+
+	"github.com/regen-network/regen-ledger/orm"
+)
+
+func TestSequence(t *testing.T) {
+	rapid.Check(t, rapid.Run(&sequenceMachine{}))
+}
+
+// sequenceMachine is a state machine model of Sequence. It simply uses a uint64
+// as the model of the sequence.
+type sequenceMachine struct {
+	ctx   orm.HasKVStore
+	seq   *orm.Sequence
+	state uint64
+}
+
+// Init sets up the real Sequence, including choosing a random initial value,
+// and intialises the model state
+func (m *sequenceMachine) Init(t *rapid.T) {
+	// Create context
+	m.ctx = orm.NewMockContext()
+
+	// Create primary key table
+	storeKey := sdk.NewKVStoreKey("test")
+	const testSeqPrefix = iota
```

Comment:
```suggestion
```

unused variable

Reply:
@robert-zaremba you resolved- but did you actually commit these suggestions? Or is there some reason this `testSeqPrefix` is still necessary?

Reply:
it looks like its used now

Reply:
I included it in other changes: https://github.com/regen-network/regen-ledger/pull/459/commits/1075cee0b52bfcbca428c831983944c0d592f370